binding/go/porcupine.go

```diff
@@ -230,11 +231,11 @@ func (porcupine *Porcupine) Delete() error {
 func (porcupine *Porcupine) Process(pcm []int16) (keywordIndex int, err error) {
 
 	if porcupine.handle == 0 {
-		return -1, fmt.Errorf("Porcupine has not been initialized or has been deleted.")
+		return -1, fmt.Errorf("Porcupine has not been initialized or has been deleted")
 	}
 
 	if len(pcm) != FrameLength {
-		return -1, fmt.Errorf("Input data frame size (%d) does not match required size of %d", len(pcm), FrameLength)
+		return -1, fmt.Errorf("input data frame size (%d) does not match required size of %d", len(pcm), FrameLength)
```

Comment:
Go expected errors to be in lower case since the usually follow other context. See: https://github.com/golang/go/wiki/CodeReviewComments#error-strings